src/cloud.c

```diff
@@ -92,38 +104,54 @@ static bool on_cloud_receive_message(const char *exchange,
 	struct l_queue *list;
 	const char *id;
 	bool consumed = true;
+	const struct event_handler *handler;
```

Comment:
As you are using a **cb_handler** maybe it would be better if this variable was declared as something like **evt_handler**

Reply:
Done